From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Javier Martinez Canillas <javierm@redhat.com>
Date: Wed, 26 Jun 2019 17:11:14 +0200
Subject: [PATCH] 10_linux: don't search for OSTree kernels

The 10_linux script used to exit if kernels weren't found in the /boot dir
so the path where OSTree kernels are installed was added to prevent that.

But the script should really add a section to call the blscfg command even
if kernels have not been installed, it doesn't make sense to search these.

Signed-off-by: Javier Martinez Canillas <javierm@redhat.com>
---
 util/grub.d/10_linux.in | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/util/grub.d/10_linux.in b/util/grub.d/10_linux.in
index e6fd6f2a39a..9fd5a16fa32 100644
--- a/util/grub.d/10_linux.in
+++ b/util/grub.d/10_linux.in
@@ -265,12 +265,6 @@ case "x$machine" in
 	done ;;
 esac
 
-if [ "x${GRUB_ENABLE_BLSCFG}" = "xtrue" ]; then
-    for i in /boot/ostree/*/vmlinuz-* ; do
-        if grub_file_is_not_garbage "$i" ; then list="$list $i" ; fi
-    done
-fi
-
 case "$machine" in
     i?86) GENKERNEL_ARCH="x86" ;;
     mips|mips64) GENKERNEL_ARCH="mips" ;;
